spec/support/jasmine-browser.json

```
@@ -2,7 +2,7 @@
   "srcDir": "public/assets/whitehall",
   "srcFiles": [
     "application-*.js",
-    "all-*.js"
+    "all_legacy-*.js"
```

I'm expecting we may want this to resemble: 

```suggestion
"admin_legacy-*.js"
"frontend-*.js"
```

since all won't mean what it used to.